Playbooks/Enrich-CIRCL-hashlookup/CustomConnector/azuredeploy.json

```diff
@@ -0,0 +1,221 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceTags": {
+            "type": "object",
+            "defaultValue": {
+                "LogicAppsCategory": "security"
+            }
+        },
+        "CustomConnectorName": {
+            "defaultValue": "CIRCL-hashlookup",
+            "type": "string",
+            "metadata": {
+                "description": "Name of the CIRCL hashlookup connector"
+            }
+        },
+        "Service EndPoint": {
+            "defaultValue": "https://hashlookup.circl.lu",
+            "type": "string",
+            "metadata": {
+                "description": ""
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        }
```

Comment:
I disagree with this per
* linter rule - bicep
https://aka.ms/bicep/linter/no-loc-expr-outside-params
https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter-rule-no-loc-expr-outside-params
* Some of my playbooks (not this one) relies on an On-Premise Data Gateway for which location must stay the same independently of the rest of the sentinel setup
* It is more aligned with the use of sentinel-deployment.config and shared parameters files. - https://learn.microsoft.com/en-us/azure/sentinel/ci-cd-custom-deploy
* It does not affect default behavior as per default value

Reply:
additional comment? can I resolve this point?

Reply:
If not providing location parameter, it will stay with resource group location.
Azure location lists will evolve over time. I think that it will be cumbersome to maintain such a list inside the ARM template unless a built-in function can provide them.

Reply:
Fair point. Thanks